### PR TITLE
Viacoin: add to lib

### PR DESCRIPTION
**Pull request by romanornr**

### CODE REVIEW

**Comment by fanatid** [comment]
Old PR: #56

**Comment by romanornr** [comment]
Sorry I messed up with git rebase -i and 2 old commits got co-authored accidentally. 
It never happened to me. After a few tries to undo it, I decided ditch the autism and just make a new pull request.

**Comment by romanornr** [comment]
Any update on this? Will it be merged ?

File: lib/coins/via.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  ],
+  versions: {
+    bip32: {
+      private: 0x05358394,
+      public: 0x0488b21e
+    },
+    bip44: 14,
+    private: 0xc7,
+    public: 0x30,
+    scripthash: 0x47
```
Comment (fanatid) [inline lib/coins/via.js]: Can you point from where you take this constants? They are not match: https://github.com/viacoin/viacoin/blob/04672d3d0a696606df0933bb307709fa23235eb4/src/chainparams.cpp#L137-L141

Comment (romanornr) [reply lib/coins/via.js]: Must have been confused. Just forced pushed an update.

File: lib/coins/via.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  port: 25223,
+  protocol: {
+    magic: 0x92efc5a9
+  },
+  seedsDns: [
+    'testnet.viacoin.net',
+    'seed-testnet.viacoin.net'
+  ],
+  versions: {
+    bip44: 1,
```
Comment (fanatid) [inline lib/coins/via.js]: bip32 versions?

Comment (romanornr) [reply lib/coins/via.js]: Just added with an amend force push

File: lib/coins/via.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    magic: 0x92efc5a9
+  },
+  seedsDns: [
+    'testnet.viacoin.net',
+    'seed-testnet.viacoin.net'
+  ],
+  versions: {
+    bip32: {
+	    private: 0x04358394,
+	    public: 0x043587cf
```
Comment (fanatid) [inline lib/coins/via.js]: padding issues

Comment (romanornr) [reply lib/coins/via.js]: yeah just noticed. Just ran ```standard --fix````

File: lib/coins/via.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  seedsDns: [
+    'testnet.viacoin.net',
+    'seed-testnet.viacoin.net'
+  ],
+  versions: {
+    bip32: {
+      private: 0x04358394,
+      public: 0x043587cf
+    },
+    bip44: 1,
```
Comment (fanatid) [inline lib/coins/via.js]: Why value here is `1`?

Comment (romanornr) [reply lib/coins/via.js]: See index 1: https://github.com/satoshilabs/slips/blob/master/slip-0044.md#registered-coin-types

Comment (romanornr) [reply lib/coins/via.js]: Any update about this?

Comment (romanornr) [reply lib/coins/via.js]: LTC and DNR have bip44 testnet as 1.

**Comment by fanatid** [comment]
Sorry for delay. Published as 4.3.0

**Pull request merged**